app/views/themes/shinycms/menu/_blog.html.erb

```diff
@@ -0,0 +1,13 @@
+<% Blog.all.each do |blog| %>
+<li>
+  <%= link_to blog.name, view_blog_path( blog ) %>
+  <ul>
+    <% menu_posts = blog.recent_posts %>
```

Comment:
Consider adding a `limit(5)` to the `blog.recent_posts` call to retrieve the only those posts that are needed from the db.